docs/apis-clients/operate-api/index.md

```diff
@@ -0,0 +1,426 @@
+| `POST /v1/flownode-instances/search`    | Search for flow node instances |
+| `GET /v1/flownode-instances/{key}`      | Get flow node instance by key |
+| **Variables**                           || 
+| `POST /v1/variables/search`             | Search for variables; results can contain truncated variable values| 
+| `GET /v1/variables/{key}`               | Get variable by key; contains the full value of variable |
+
+## Search
+
+Every object has a search `/v1/<object>/search` endpoint which can be requested by `POST` and a given query request.
+
+### Query
+
+The query request consists of components for **filter**, **size**, **sort**, and **pagination**.
+
+```
+{
+   "filter": { object fields to match },
+   "size": <number of items to return>,
+   "sort": [ {"field":"<name of field to sort on>", "order": "<ASC|DESC>" ],
+   "searchAfter": [ <identifier of item from which next search should start> ] 
+}
+```
+
+#### Filter
+
+Specifies which fields should match. Only items that match the given fields will be returned.
+The section on [object schemas](#object-schemas) lists all available fields for each object.
+
+##### Filter strings, numbers, and booleans
+
+Fields of type string, number, and boolean need the exact value to match.
+
+###### Examples
+
+Return all items with field `processInstanceKey` equals `235`:
+
+```json
+{ "filter": { "processInstanceKey": 235 } }
+```
+
+Return all items with field `processInstanceKey` equals `235`, `state` equals `ACTIVE` and `incidents` equals `true`:
+
+```json
+{ "filter": { "processInstanceKey": 235, "state": "ACTIVE", "incidents": true }}
+```
+
+##### Filter dates
+
+Date fields need to be specified in format: `yyyy-MM-dd'T'HH:mm:ss.SSSZZ`; for example, `2022-03-17T11:50:25.729+0000`.
+
+You can use modifier to match date ranges:
+
+| Modifier | Description |
+|--|-------------|
+| \|\|/y  | Within a year |
+| \|\|/M | Within a month |
+| \|\|/w | Within a week |
+| \|\|/d | Within a day |
+| \|\|/h | Within an hour |
+| \|\|/m | Within a minute |
+| \|\|/s | Within a second |
+
+###### Example
+
+Return all items with field `startDate` within a minute (`||/m`) for `2022-03-17 11:50:25`.
+
+```json
+{
+  "filter": {
+    "startDate": "2022-03-17T11:50:25.729+0000||/m"
+  }
+}
+```
+
+#### Size
+
+Maximum items should be returned and must be a number.
+
+##### Example
+
+Return maximum `23` items:
+
+```json
+{ "size": 23 }
+```
+
+#### Sort
+
+Specify which field of the object should be sorted and whether ascending (`ASC`) or descending (`DESC`).
+
+##### Example
+
+Sort by `name` **desc**ending:
+
+```json
+{ "sort": [{"field":"name","order":"DESC"}] }
+```
+
+#### Pagination
+
+Specify the item where the next search should start. For this, you need the values from previous results. 
+Copy the values from `sortValues` field from the previous results into the `searchAfter` value of query. 
+See also [results](#results).
+
+##### Example
+
+Get next page for sorted by name descending:
```

Comment:
What do you mean here?

Reply:
Thanks, @christinaausley for the corrections.

I want to show the mechanism to get the next results for the previously described query. I changed the description in another commit. I hope it is now clearer.